tests/data/microsemitime.json

```diff
@@ -0,0 +1,1830 @@
+{
+    "os": {
+        "discovery": {
+            "devices": [
+                {
+                    "sysName": "<private>",
+                    "sysObjectID": ".1.3.6.1.4.1.39165.1.6",
+                    "sysDescr": "Linux SyncServer 2.6.10_mvl401-pc_target.Gryphon.2.10.2009.1201126 #18 Thu Mar 10 10:31:17 PST 2016 i686",
+                    "sysContact": null,
+                    "version": null,
+                    "hardware": null,
+                    "features": null,
+                    "os": "microsemitime",
+                    "type": "timing",
+                    "serial": null,
```

Comment:
you need to generate new test data, as you can see here your version/serial/hardware is empty